### PR TITLE
Fix: PHP Lint fixes [ED-3168, ED-2901]

### PR DESCRIPTION
**Pull request by guyc-E**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Comment by guyc-E** [comment]
Lint / PHP-Lint (pull_request) - Fails on files I didn't changed.

File: modules/library/traits/library.php [hunk 1]
```diff
@@ -21,8 +21,8 @@ trait Library {
 	 */
 	public function print_admin_column_type() {
 		$admin_filter_url = admin_url( Source_Local::ADMIN_MENU_SLUG . '&elementor_library_type=' . $this->get_name() );
-
-		printf( '<a href="%s">%s</a>', $admin_filter_url, $this->get_title() );
+		//PHPCS - Not user input
```
Comment (kobizz) [inline modules/library/traits/library.php:24]:
```suggestion
		//PHPCS - Not a user input
```

File: modules/library/traits/library.php [hunk 2]
```diff
@@ -21,8 +21,8 @@ trait Library {
...
+		printf( '<a href="%s">%s</a>', esc_url( $admin_filter_url ), $this->get_title() ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```
Comment (kobizz) [inline modules/library/traits/library.php:25]: You don't need to escape this url

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by kobizz — APPROVED** [review]

File: phpcs.xml [hunk 3]
```diff
@@ -26,16 +26,10 @@
 		<exclude name="Squiz.PHP.EmbeddedPhp.ContentBeforeEnd" />
 		<exclude name="Squiz.PHP.EmbeddedPhp.ContentAfterEnd" />
 		<exclude name="WordPress.Arrays.MultipleStatementAlignment" />
-		<exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification" />
```
[the remark this answers is not in the document]
Comment (matipojo) [reply phpcs.xml:29]: It's OK the final rules will be set later.

**Pull request merged**